templates/kubernetes/features.html

```diff
@@ -0,0 +1,160 @@
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Multi-cloud workload portability thanks to Ubuntu</li>
+        <li class="p-list__item is-ticked">Same Ubuntu OS and kernel as Google, Azure, AWS k8s offerings</li>
+        <li class="p-list__item is-ticked">PCI device passthrough for GPGPU, FPGA and SR-IOV workloads</li>
+        <li class="p-list__item is-ticked">AI and machine learning optimisations with nVidia and Google</li>
+        <li class="p-list__item is-ticked">Flavours to match public cloud k8s on Google, Azure, AWS</li>
+        <li class="p-list__item is-ticked">Standard upstream Kubernetes, CNCF builds available</li>
+        <li class="p-list__item is-ticked">Portfolio of third-party certified solutions for CI/CD</li>
+        <li class="p-list__item is-ticked">Day-2 operations encapsulated and included</li>
+        <li class="p-list__item is-ticked">Upgrades guaranteed between K8s releases</li>
+        <li class="p-list__item is-ticked">Security updates by Canonical just like Ubuntu</li>
+        <li class="p-list__item is-ticked">Full OCI compatibility with docker and containerd runtimes</li>
+        <li class="p-list__item is-ticked">Prometheus and ELK logging and monitoring included</li>
+        <li class="p-list__item is-ticked">Extensible, third-party ecosystem for storage, networking</li>
+      </ul>
+      <p>In partnership with Google, Canonical delivers the perfect K8s for AI and machine learning with Kubeflow.</p>
+      <p><a href="/ai/kubeflow">Learn more about Kubeflow&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Networking</h2>
+      <p>Kubernetes imposes fundamental requirements on your networking implementation including:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">all containers can communicate with all other containers without NAT</li>
+        <li class="p-list__item is-ticked">all nodes can communicate with all containers (and vice-versa) without NAT</li>
+        <li class="p-list__item is-ticked">the IP that a container sees itself as is the same IP that others see it as</li>
+      </ul>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p>Kubernetes uses Container Network Interface (CNI) as an interface between network providers and Kubernetes networking. CNI is a library definition and a set of tools, under the umbrella of the Cloud Native Computing Foundation project.</p>
+      <p>Canonical&rsquo;s Distribution of Kubernetes comes pre-packaged with several tested CNI plugins like Calico and Flannel. Our distribution of Kubernetes is open and extensible &mdash; bring your favorite CNI plugin and extend CDK. If you require assistance, please take advantage of one of our consulting packages.</p>
```

Comment:
Looks like the sentence "If you require assistance, please take advantage of one of our consulting packages." has been removed from the copy doc.

Reply:
Done